autogpt/config/ai_config.py

```diff
@@ -114,6 +114,8 @@ def construct_full_prompt(
             " simple strategies with no legal complications."
             ""
         )
+        os_name = platform.system()
```

Comment:
From the `platform` docs:
`platform.system`: `Returns the system/OS name, such as 'Linux', 'Darwin', 'Java', 'Windows'.`

`Darwin` refers to MacOS, which is a much more common term; outputting that probably results in better behavior of the AI. You can either map this manually, or use e.g. `platform.platform(terse=True)`

Reply:
Thanks for pointing it out, just commited the changes.